cms/envs/devstack-experimental.yml

```diff
@@ -314,6 +314,7 @@ FEATURES:
     ENABLE_ENROLLMENT_RESET: false
     ENABLE_EXPORT_GIT: false
     ENABLE_GRADE_DOWNLOADS: true
+    ENABLE_LIBRARY_AUTHORING_MICROFRONTEND: true
```

Comment:
@connorhaugh: Is it okay if I turn on v2 content libraries in devstack by default? Or should I leave it at v1 libraries, and just override this locally? (How do you override this locally?)